src/main/preload/mattermost.js

```diff
@@ -149,13 +149,6 @@ window.addEventListener('message', ({origin, data = {}} = {}) => {
         ipcRenderer.send(BROWSER_HISTORY_BUTTON, viewName);
         break;
     }
-    default:
```

Comment:
These logs generate a LOT of spam when developing and made it hard to read the logs you want to read. So I've removed them.

Reply:
totally fine, those were to catch any unexpected message coming from the webapp. It's been a while since then so it shouldn't happen or it might be safely ignored.